includes/admin/views/html-admin-page-status-report.php

```diff
@@ -528,6 +528,9 @@
 				if ( ! $page_id ) {
 					echo '<mark class="error">' . __( 'Page not set', 'woocommerce' ) . '</mark>';
 					$error = true;
+				} else if ( get_post_status ( $page_id ) != 'publish' ) { // make sure the page isn't private or a draft
+					echo '<mark class="error">' . sprintf( __( 'Page visibility should be <a href="%s" target="_blank">public</a>', 'woocommerce' ), 'https://codex.wordpress.org/Content_Visibility' ) . '</mark>';
```

Comment:
Remove HTML from string to translate. Use %s before and %s after for open/close anchor.